### PR TITLE
Add SIGDIAL / INLG DOIs

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/xml/2022.inlg.xml [hunk 1]
```diff
@@ -454,7 +479,8 @@
...
+      <abstract>This paper describes the system description for the HinglishEval challenge at INLG 2022. The goal of this task was to investigate the factors influencing the quality of the code- mixed text generation system. The task was divided into two subtasks, quality rating pre- diction and annotators’ disagreement predic- tion of the synthetic Hinglish dataset. We at- tempted to solve these tasks using sentence- level embeddings, which are obtained from mean pooling the contextualized word embed- dings for all input tokens in our text. We experimented with various classifiers on top of the embeddings produced for respective tasks. Our best-performing system ranked 1st on subtask B and 3rd on subtask A. We make our code available here: <url>https://github.com/nikhilbyte/Hinglish-qEval</url>
+      </abstract>
```
Comment (mbollmann) [inline data/xml/2022.inlg.xml:483]: Weird that this got reformatted

**Review by mbollmann — APPROVED** [review]
Spot-checked a few, LGTM

File: data/xml/2022.inlg.xml (hunk 1 above)
Comment (akoehn) [reply data/xml/2022.inlg.xml:483]: Maybe you can amend the commit and do a force push to undo this @mjpost?

Comment (mjpost) [reply data/xml/2022.inlg.xml:483]: I just fixed it, along with a handful of hyphenations.

Comment (mjpost) [reply data/xml/2022.inlg.xml:483]: Actually, grepping for the pattern, it seems that any time an abstract ends with `</url>`, the abstract closing tag got pushed to the next line. I wonder if this was introduced in #2806.

**Comment by mjpost** [comment]
See #2888

**Comment by mdingemanse** [comment]
Thanks so much for this!

**Pull request merged**